Station/pycpt-seasonal-station-grid-Chile.ipynb

```diff
@@ -0,0 +1,794 @@
+    "    #Grid data 0.25 1960-2021\n",
+    "    'CR2MET0p25.PRCP': \"https://iridl.ldeo.columbia.edu/SOURCES/.Chile/.DMC/.0p05/.monthly/.prcp/X/first/0.25/last/GRID/Y/first/0.25/last/GRID/T/(1%20Jan%20{first_year})/(1%20Dec%20{final_year})/RANGE/T/%28{target}%20{first_year}-{final_year}%29/seasonalAverage/Y/%28{predictand_extent['south']}%29/%28{predictand_extent['north']}%29/RANGEEDGES/X/%28{predictand_extent['west']}%29/%28{predictand_extent['east']}%29/RANGEEDGES/-999/setmissing_value/{'%5BX/Y%5D%5BT%5Dcptv10.tsv' if filetype == 'cptv10.tsv' else 'data.nc'}\",\n",
+    "    'CR2MET0p25.TMAX': \"https://iridl.ldeo.columbia.edu/SOURCES/.Chile/.DMC/.0p05/.monthly/.tmax/X/first/0.25/last/GRID/Y/first/0.25/last/GRID/T/(1%20Jan%20{first_year})/(1%20Dec%20{final_year})/RANGE/T/%28{target}%20{first_year}-{final_year}%29/seasonalAverage/Y/%28{predictand_extent['south']}%29/%28{predictand_extent['north']}%29/RANGEEDGES/X/%28{predictand_extent['west']}%29/%28{predictand_extent['east']}%29/RANGEEDGES/-999/setmissing_value/{'%5BX/Y%5D%5BT%5Dcptv10.tsv' if filetype == 'cptv10.tsv' else 'data.nc'}\",\n",
+    "    'CR2MET0p25.TMIN': \"https://iridl.ldeo.columbia.edu/SOURCES/.Chile/.DMC/.0p05/.monthly/.tmin/X/first/0.25/last/GRID/Y/first/0.25/last/GRID/T/(1%20Jan%20{first_year})/(1%20Dec%20{final_year})/RANGE/T/%28{target}%20{first_year}-{final_year}%29/seasonalAverage/Y/%28{predictand_extent['south']}%29/%28{predictand_extent['north']}%29/RANGEEDGES/X/%28{predictand_extent['west']}%29/%28{predictand_extent['east']}%29/RANGEEDGES/-999/setmissing_value/{'%5BX/Y%5D%5BT%5Dcptv10.tsv' if filetype == 'cptv10.tsv' else 'data.nc'}\",\n",
+    "    'CR2MET0p25.TMEAN': \"https://iridl.ldeo.columbia.edu/SOURCES/.Chile/.DMC/.0p05/.monthly/.tmean/X/first/0.25/last/GRID/Y/first/0.25/last/GRID/T/(1%20Jan%20{first_year})/(1%20Dec%20{final_year})/RANGE/T/%28{target}%20{first_year}-{final_year}%29/seasonalAverage/Y/%28{predictand_extent['south']}%29/%28{predictand_extent['north']}%29/RANGEEDGES/X/%28{predictand_extent['west']}%29/%28{predictand_extent['east']}%29/RANGEEDGES/-999/setmissing_value/{'%5BX/Y%5D%5BT%5Dcptv10.tsv' if filetype == 'cptv10.tsv' else 'data.nc'}\"\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "076ded6b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MOS = 'CCA' # must be one of 'CCA', 'PCR'\n",
+    "\n",
+    "# Use dl.observations.keys() to see all options for predictand \n",
+    "# and dl.hindcasts.keys() to see all options for predictors.\n",
+    "# Make sure your first_year & final_year are compatible with \n",
+    "# your selections for your predictors and predictands.\n",
+    "predictor_names = [ \"CanSIPSIC3.PRCP\", 'SEAS51.PRCP']\n",
```

Comment:
SEAS51 is deprecated. During download, warnings to that effect appear. Switch to SEAS51b.PRCP.